### PR TITLE
add pages missing to reference overview

### PR DESCRIPTION
**Pull request by akeller**
## Description

New(ish) pages were missing from Reference overview page. I added a new "Policies" header and tried to group these as they made sense. This may not be the best grouping. 

Please comment or add a review if you have better suggestions for groupings, headers, or other info. 

This should be backported. The sidebar should also reflect the final ordering on this page (groupings via headers may not be necessary at this time).

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@christinaausley I'm looking for some collaboration here. Check out the description and the changes and let me know your thoughts.

**Review by christinaausley — COMMENTED** [review]
I added descriptions, but I have a few questions we can work to answer together:

- Is this listed in sections by most to least important? Users spend limited time on documentation per page, so I just want to ensure what is most important or what we would want users to visit first is in order. Based on this, it could be useful to group the sidebars by this organization.

- In announcements, we say the Web Modeler beta API was deprecated, and while beta has been removed, I think this is misleading as the API still very much exists. I think we can reword this to ensure users know they can utilize the API -- do you agree?

- Is the early access page a policy? It seems more like something that would go under support. If we did this, I would likely just add an admonition for policies and link to the release policy below the security and license information.

**Comment by akeller** [comment]
> In announcements, we say the Web Modeler beta API was deprecated, and while beta has been removed, I think this is misleading as the API still very much exists. I think we can reword this to ensure users know they can utilize the API -- do you agree?

@christinaausley This isn't in the scope of this PR, but I agree this could be reworded to reduce the risk of confusion. Could you open another PR for this and loop in @christian-konrad?

**Comment by akeller** [comment]
> Is this listed in sections by most to least important? Users spend limited time on documentation per page, so I just want to ensure what is most important or what we would want users to visit first is in order. Based on this, it could be useful to group the sidebars by this organization. Or, we can always just alphabetize.

This is my opinionated approach - most to least important by section. I do think the sidebar should match the page order, but not necessarily the groupings. 

> Is the early access page a policy? It seems more like something that would go under support. If we did this, I would likely just add an admonition for policies and link to the release policy below the security and license information.

Yes and no? I like your suggestion - I will update the PR.

**Comment by christinaausley** [comment]
I'm good with this @akeller -- any additional adjustments I can make?

**Comment by akeller** [comment]
Backport and it is ready to go!

**Review by christinaausley — APPROVED** [review]
